onnxruntime/python/tools/tensorrt/perf/benchmark.py

```diff
@@ -419,7 +352,15 @@ def generate_onnx_model_random_input(test_times, ref_input):
 
     return inputs
 
-def validate(all_ref_outputs, all_outputs, rtol=0, atol=1.5):
+def percentage_in_allowed_threshold(e, percent_mismatch):
```

Comment:
I'm assuming that this threshold check function is added for mismatch output we got from some "object detection" models, correct?
In the long run it's better for us to add different validation approaches for different kind of models, otherwise this generic threshold check function may overlook some errors.
But I think for now this function is okay.